src/openms/source/ANALYSIS/OPENSWATH/MRMDecoy.cpp

```diff
@@ -537,9 +537,9 @@ namespace OpenMS
     // Go through all peptides and apply the decoy method to the sequence
     // (pseudo-reverse, reverse or shuffle). Then set the peptides and proteins of the decoy
     // experiment.
-    for (Size i = 0; i < exp.getPeptides().size(); i++)
+    for (Size pep_it = 0; pep_it < exp.getPeptides().size(); ++pep_it)
```

Comment:
Pep_idx is fine with me.
For simple loops (not nested ) i'd still prefer i.
But I can totally see your point. 
-- 
Sent via mobile phone.

Am 17. August 2015 22:01:14 MESZ, schrieb Hannes Roest <notifications@github.com>:
>> @@ -537,9 +537,9 @@ namespace OpenMS
>>      // Go through all peptides and apply the decoy method to the
>sequence
>>      // (pseudo-reverse, reverse or shuffle). Then set the peptides
>and proteins of the decoy
>>      // experiment.
>> -    for (Size i = 0; i < exp.getPeptides().size(); i++)
>> +    for (Size pep_it = 0; pep_it < exp.getPeptides().size();
>++pep_it)
>
>also I believe that naming the variable about regarding to which vector
>it indexes makes more sense than just `i`, but maybe `pep_idx` makes
>more sense than `pep_it` - what do you think?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/OpenMS/OpenMS/pull/1547/files#r37229983